qcodes/tests/test_parameter.py

```diff
@@ -1192,11 +1192,26 @@ class TestSetContextManager(TestCase):
 
     def setUp(self):
         self.instrument = DummyInstrument('dummy_holder')
-        self.instrument.add_parameter(
-            "a",
-            set_cmd=None,
-            get_cmd=None
-        )
+
+        self._vp_value = "foo"
```

Comment:
could you move this line close to the "validated_param" parameter so that it's easier to notice? it took me a while to understand why _vp_getter and _vp_setter are needed :)